app/settings/surveys/survey-editor.directive.js

```diff
@@ -472,10 +472,61 @@ function SurveyEditorController(
             ModalService.close();
         }
         $scope.editAttribute = attribute;
-        var title = attribute.id ? 'survey.edit_field' : 'survey.add_field';
+        var fieldType = getFieldType(attribute) ? getFieldType(attribute) : attribute.label;
+        var title = attribute.id ? $translate.instant('survey.edit_field', {fieldType: fieldType}) : 'survey.add_field';
```

Comment:
Could you add the type of field when adding a new field as well? Same method as when editing, just add the translate-values to "survey.add_field" as well.